java/java.sourceui/src/org/netbeans/api/java/source/ui/ElementJavadoc.java

```diff
@@ -1280,6 +1282,22 @@ private StringBuilder inlineTags(List<? extends DocTree> tags, TreePath docPath,
                 case TEXT:
                     TextTree ttag = (TextTree)tag;
                     sb.append(ttag.getBody());
+                    break;
+                default : {
+                    // process tags that we cannot add directly because they are not accessible during compilation
+                    DocTree.Kind values[] = tag.getKind().getClass().getEnumConstants();
```

Comment:
For consideration here, how about, instead of finding the summaryTag, there would be just `if ("SUMMARY".equals(tag.getKind().name())) ..` or `switch (tag.getKind().name()) { case "SUMMARY":`?

Reply:
I will change it in accordance with your proposal. But I don't know how to add unit test that is executed only with JDK 11 for example. Is there any example how to do it?

Reply:
I added unit test, but need to change some additional files because I have to set correct source level.